incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -246,6 +291,7 @@ func CleanupNamespaces(nses ...string) {
 	for _, ns := range nses {
 		// Skip any namespace that doesn't actually exist. We only check once (e.g. no retries on
 		// errors) but reads are usually pretty reliable.
+
```

Comment:
Was this intentional? This applies to the line 295-298, not to the whole block, so I wouldn't put a blank space here.

Also, if we've just finished searching for labels, we don't need this check at all.

Reply:
Yes we dont need this block if its only called from CleanupTestNamespaces which search for labels. But someone does , it won't gurantee the the ns exists. I can remove this .

Reply:
I tried removing the check for the existing namespaces as it is not relevant if we filter by labels anyway as you pointed out. But there are some connected tests which fail, so i restored it back. There is a function RecoverHNC which creates and deletes specific namespaces only. Deleting all namespaces causes to delete a namespace called 'child' in a previous test causing all tests to fail. Let me know if you have any suggestions to fix it or if it can be tracked seperately

Reply:
sgtm